_includes/compact_hub_cards.html

```diff
@@ -8,7 +8,15 @@
         <a href="{{ site.baseurl }}{{ item.url }}">
           <div class="compact-card-body">
             <div class="compact-card-container">
-              <h4 class="compact-item-title col-6">{{ item.title }}</h4>
+              <div class="compact-hub-card-title-container">
+              <h4 class="compact-item-title">{{ item.title }}</h4>
+              <div class="detail-stars-container col-2">
+                <div class="github-stars-image">
+                  <img src="/assets/images/github-star.svg">
+                </div>
+                <div class="github-stars-count"></div>
+              </div>
+              </div>
```

Comment:
nit - can you double-check the indentation here? It looks like the `detail-stars-container` div needs to be indented.